app/views/AdminPanelView/index.js

```diff
@@ -15,7 +15,7 @@ import styles from '../Styles';
 	authToken: state.login.user && state.login.user.token
 }))
 /** @extends React.Component */
-export default class AdminPanelView	 extends LoggedView {
+export default class AdminPanelView	extends LoggedView {
```

Comment:
It's nitpicking, but it's so easy to fix that I need to request a change: there's a tab here.
Change for a normal space.

Reply:
my vs code settings just changed and  have tab length equals to 1 space